src/modules/commander/state_machine_helper.cpp

```diff
@@ -310,11 +310,9 @@ main_state_transition(struct vehicle_status_s *status, main_state_t new_main_sta
 		break;
 
 	case vehicle_status_s::MAIN_STATE_ALTCTL:
-		/* need at minimum altitude estimate */
-		/* TODO: add this for fixedwing as well */
-		if (!status->is_rotary_wing ||
```

Comment:
A lot of people are actually flying multicopter without GPS, because holding altitude is hard. The longer I look at this the more I have the impression the right fix is to prevent arming.